src/blog/2023/06/introducing-the-flowforge-community-forum.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Introducing the FlowForge Community Forum
+subtitle: A Community Forum for support, inspiration, and knowledge sharing
+description: 
+date: 2023-06-09 12:00:00
+authors: ["zeger-jan-van-de-weg"]
+tags:
+  - news
+  - flowforge
+image: # /images/blog/tile-needed.jpg
```

Comment:
@iskerrett I like the generated tile, and don't want to wait on a "nice" tile. I'd be fine with merge with this tile.